src/sentry/tagstore/base.py

```diff
@@ -89,6 +89,8 @@ class TagStorage(Service):
         'incr_group_tag_value_times_seen',
         'update_group_tag_key_values_seen',
         'update_group_for_events',
+
+        'delay_index_event_tags',
```

Comment:
This doesn't actually do any actual writes but it's closest to the write API so I put it here